src/SVG.gd

```diff
@@ -109,6 +112,9 @@ func apply_svg_from_path(path: String) -> int:
 	var error := ""
 	var extension := path.get_extension()
 	
+	GlobalSettings.current_file_name = path.get_file().rstrip("." + path.get_extension())
```

Comment:
Use `remove_suffix()` instead.

Reply:
ok but what's the difference

Reply:
Something like "my_svg.svg" will get stripped to "my_"